src/participant_network.star

```diff
@@ -149,14 +149,32 @@ def launch_participant_network(
             network_params.deneb_fork_epoch,
             network_params.electra_fork_epoch,
         )
+    elif network_params.network in constants.PUBLIC_NETWORKS:
+        dummy = plan.run_sh(
+            run="mkdir /network-configs",
+            store=[StoreSpec(src="/network-configs/", name="el_cl_genesis_data")],
+        )
+        el_cl_data = el_cl_genesis_data.new_el_cl_genesis_data(
+            dummy.files_artifacts[0],
+            "0",
+        )
+
+        final_genesis_timestamp = 0
+        validator_data = None
     else:
         # split up the name from dencun-devnet-12 to dencun and devnet-12
-        devnet_name = network_params.network.split("-")[0]
-        devnet_number = network_params.network.split("-")[-1]
+        # split up the name from verkle-gen-devnet-12 to verkle-gen and devnet-12
+        network_parts = network_params.network.split("-devnet-", 1)
+        devnet_name, devnet_number = network_parts[0], network_parts[1]
+        devnet_category = devnet_name.split("-")[0]
+        devnet_subname = (
+            devnet_name.split("-")[1] + "-" if len(devnet_name.split("-")) > 1 else ""
+        )
+        url = "github.com/ethpandaops/{0}-devnets/network-configs/{1}devnet-{2}".format(
```

Comment:
you can take the url format as a constant up top

Reply:
I would only want to "calculate" this url constant if I'm running a devnet

Reply:
Fixed